src/Creatable.js

```diff
@@ -151,6 +151,14 @@ const Creatable = React.createClass({
 		});
 	},
 
+	componentWillUpdate (nextProps, nextState) {
+		// Tricky: Async clears the select inputValue when deleting
+		// an option, but it doesn't fire onInputChange.
```

Comment:
Right, but I guess I don't understand what the comment has to do with this new bit of code? Why does not firing onInputChange matter to setting the inputValue?